source/MRViewer/ImGuiMenu.h

```diff
@@ -45,6 +48,12 @@ class MRVIEWER_CLASS ImGuiMenu : public MR::ViewerPlugin,
   // if true, then pre_draw will start from polling glfw events
   bool pollEventsInPreDraw = false; // be careful here with true, this can cause infinite recurse 
 
+  bool showStatistics_{ false };
+  long long frameTimeMillisecThreshold_{ 25 };
+  bool show_rename_modal_{ false };
```

Comment:
please make functions to change `showStatistics_` and `show_rename_modal_` (also pls rename it to `showRenameModal_` ) and remove helpers code from `MR::Menu`
P.S. `F1` and `D` button should change `showRenameModal_` and `showStatistics_`, but from `RibbonMenu`